src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -199,37 +195,18 @@ public override void Open()
                     break;
             }
 
-            if ((flags & SQLITE_OPEN_URI) == 0
-                && !filename.Equals(":memory:", StringComparison.OrdinalIgnoreCase)
-                && !Path.IsPathRooted(filename))
-            {
-                filename = Path.GetFullPath(Path.Combine(BaseDirectory, filename));
-            }
-
-            var rc = NativeMethods.sqlite3_open_v2(filename, out _db, flags, vfs: null);
-            MarshalEx.ThrowExceptionForRC(rc, _db);
+            var rc = raw.sqlite3_open_v2(filename, out _db, flags, vfs: null);
+            SqliteException.ThrowExceptionForRC(rc, _db);
 
             SetState(ConnectionState.Open);
         }
 
-#if !NET451
-        private static string BaseDirectory
-            => Environment.GetEnvironmentVariable("ADONET_DATA_DIR")
```

Comment:
Don't we need this for EF migrations to keep working? cref https://github.com/aspnet/EntityFramework.Tools/blob/678ab9a85aa804500c874b175a0000b277de4a12/src/ef/OperationExecutorBase.cs#L14